### ISSUE TITLE
Remove the username and password option

### ISSUE THREAD

**Issue by gbockus** [gbockus/PR-Chome-Ext#3]

**Comment by jeremy-shannon** [comment]
This issue is handled by the PR I just sent. Low hanging fruit. 

I suppose I can't close the issue myself?

**Comment by gbockus** [comment]
Yeah maybe not, but if I make you a contributor I think you can.  The easiest way to doc a PR in an issue is to include the issue number in your pr title or the commits.  #3 in this case would have made it show up here.

**Comment by jeremy-shannon** [comment]
Good to know. Thanks. 

If you think it'd be best to NOT make me a contributor, because I might close an issue prematurely or whatever, that's totally fine with me.
